### DISCUSSION TITLE
Security Onion Installation Fails at 95%. Throws `certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0` Error

### DISCUSSION THREAD

**Discussion by 0xabdi** [Security-Onion-Solutions/securityonion#7003] [General] [answered]
Here is the error it throws:

```
ubuntu@sensor:~/securityonion$ sudo cat /root/sosetup.log | grep ERROR
[ERROR   ] Failed to pull somanager:5000/security-onion-solutions/so-nginx:2.3.91: Error 500: Get "https://so2-manager:5000/v2/": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
```

Installation Steps I undertook:
```
1. git clone https://github.com/Security-Onion-Solutions/securityonion
2. cd securityonion
3. sudo bash so-setup-network
```

install type : "DISTRIBUTED"
node type : "SENSOR"

The error is thrown on the sensor where I want to do the installation. The sensor is running on an Ubuntu EC2 VM on AWS. The manager is hosted on an on-prem location. The connection between the two has been established over an openVPN connection.

**Comment by dougburks** [comment, marked as answer]
This looks similar to https://github.com/Security-Onion-Solutions/securityonion/discussions/6997#discussioncomment-2065776.

What version of Ubuntu are you running on the manager and sensor?

**Reply by 0xabdi** [reply]
Yes, it is similar #6997 

We are running ubuntu 20 on the sensor and ubuntu 18 on the manager. Looks like the issue is related to `openssl` mismatch as mentioned on #6997 

We are downgrading the sensor to ubuntu 18

**Reply by dougburks** [reply]
I've added a warning to the documentation:
https://docs.securityonion.net/en/2.3/installation.html

**Reply by 0xabdi** [reply]
Alright,

Thanks @dougburks
